tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -44,6 +45,7 @@ import {
   screen,
   within,
 } from 'tests/unit/helpers';
+import { MZA_LAUNCH_DATETIME } from 'amo/utils/fxa';
```

Comment:
Why importing this constant here given you imported everything on line 6? (in `MzAUtils`)